libr/bin/mangling/microsoft_demangle.c

```diff
@@ -134,6 +134,33 @@ static EDemanglerErr parse_function(const char *sym, STypeCodeStr *type_code_str
 static EDemanglerErr parse_microsoft_mangled_name(const char *sym, char **demangled_name, size_t *chars_read);
 static EDemanglerErr parse_microsoft_rtti_mangled_name(const char *sym, char **demangled_name, size_t *chars_read);
 
+static const char *str_skip_prefix(R_NONNULL const char *s, R_NONNULL const char *prefix) {
+	if (r_str_startswith (s, prefix)) {
+		s += strlen (prefix);
+	}
+	return s;
```

Comment:
i think it makes more sense to return null here

Reply:
you can move this function into r_util/r_str.h as `inline` so you can use it anywhere in r2land without adding a new symbol.. we will change that in 5.8

Reply:
> i think it makes more sense to return null here

This is supposed to be more like "remove if it exists, otherwise I don't care", like
- Python `str.removeprefix()`
- Go `strings.TrimPrefix()`

but if that doesn't make sense will fix

> you can move this function into r_util/r_str.h as inline so you can use it anywhere in r2land without adding a new symbol.. we will change that in 5.8

will do